nautobot/core/ui/object_detail.py

```diff
@@ -1841,8 +1841,8 @@ def render_value(self, key, value, context: Context):
         elif cf.type == CustomFieldTypeChoices.TYPE_JSON and value is not None:
             return format_html(
                 """<p>
-                    <button class="btn btn-xs btn-primary" type="button" data-toggle="collapse"
-                            data-target="#cf_{field_key}" aria-expanded="false" aria-controls="cf_{field_key}">
+                    <button class="btn btn-sm btn-primary" type="button" data-bs-toggle="collapse"
+                            data-bs-target="#cf_{field_key}" aria-expanded="false" aria-controls="cf_{field_key}">
```

Comment:
This looks like the correct fix. Any idea why #8237 is showing up as a part of the diff set as well?